frontend/src/components/census/CensusModalComponent.tsx

```diff
@@ -18,20 +18,39 @@ function CensusModalComponent({
   show,
   onHide,
   campaignTitle,
-  campaignDescription,
   censusId,
-  censusQuestions,
 }: CensusModalProps) {
   const { userPin } = useAuth();
+  const [campaignDetails, setCampaignDetails] = useState<any>([]);
   const [currentStep, setCurrentStep] = useState(0);
   const [userData, setUserData] = useState<UserData>({
     userPin,
     campaignId: censusId,
     censusAnswers: [],
   });
 
+  const fetchCampaignDetails = async () => {
+    try {
+      const response = await axios.get(
+        `${API_URLS.ACTIVE_CAMPAIGNS}/${censusId}/CENSUS`
```

Comment:
census with small letters?

Reply:
We'll comment on that with the backend and decide for the next PR